### PR TITLE
fix reference to plugins in metadata

### PR DESCRIPTION
**Pull request by kecnry**
### Description

This pull request fixes internal references to plugin names which changed to be the human-friendly name in #2680 (not yet released), but missed updating a few cases, resulting in moment maps not being linked correctly.

### Change log entry

- [x] Is a change log needed? If yes, is it added to `CHANGES.rst`? If you want to avoid merge conflicts,
  list the proposed change log here for review and add to `CHANGES.rst` before merge. If no, maintainer
  should add a `no-changelog-entry-needed` label.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Did the CI pass? If not, are the failures related?
- [x] Is a milestone set? Set this to bugfix milestone if this is a bug fix and needs to be released ASAP; otherwise, set this to the next major release milestone. Bugfix milestone also needs an accompanying backport label.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py
Comment (pllim) [inline jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py:131]: I confirmed locally that this test fails on `main` without this patch as follows:

```
E       AssertionError: assert ('Pixel Axis 0 [z]' == 'Pixel Axis 0 [z]'
E
E           Pixel Axis 0 [z] and 'Pixel Axis 0 [y]' == 'Pixel Axis 1 [x]'
E
E         - Pixel Axis 1 [x]
E         ?            ^  ^
E         + Pixel Axis 0 [y]
E         ?            ^  ^)
```

**Review by pllim — APPROVED** [review]
Wow, the fix is very subtle. Thanks for hunting it down. I added a check as regression test. I think this is good to merge once CI is green. No need for second review.

**Comment by kecnry** [comment]
Thanks for tracking down the offending PR and adding the test! 🎉

**Pull request merged**
